### DISCUSSION TITLE
Localization support in Designer

### DISCUSSION THREAD

**Discussion by johnmorr-citrix** [elsa-workflows/elsa-core#3019] [General]
Does the current stable release (2.7) support a designer UI that can be localized?

**Comment by sfmskywalker** [comment]
It does, but to an extent - not all components are localized yet.

To localize the 2.7 designer, look for `localizations.ts` files, add translations, and submit a PR.

This isn't the ideal way though - I should have kept the translation bundles separate from the designer project.
For Elsa 3, it will be done this way from the get go.

**Reply by johnmorr-citrix** [reply]
How does it handle localization of custom activities names, properties, etc?
